packages/interface/src/components/more-menu-feature-toggle/index.js

```diff
@@ -10,9 +10,10 @@ import { speak } from '@wordpress/a11y';
 /**
  * Internal dependencies
  */
-import { store as editWidgetsStore } from '../../store';
+import { store as interfaceWidgetsStore } from '../../store';
```

Comment:
Shouldn't this be just `interfaceStore`?

Reply:
Hmm, yeah 😄

Reply:
Fixed in 34d2f2a